export const handleUncaughtExceptionMonitor = (error) => {
  console.log(`[test] uncaught exception ${error}`)
}
